internal/modifier/cdi.go

```diff
@@ -116,6 +117,24 @@ func getDevicesFromSpec(logger logger.Interface, ociSpec oci.Spec, cfg *config.C
 	return nil, nil
 }
 
+func normalizeDeviceList(logger logger.Interface, defaultKind string, devices ...string) []string {
```

Comment:
Maybe add a code comment showing an example of what this function is doing (supposed to do).